editor/components/inserter/menu.js

```diff
@@ -207,7 +208,7 @@ export class InserterMenu extends Component {
 							<PanelBody
 								key={ category.slug }
 								title={ category.title }
-								opened={ isPanelOpen( category.slug ) }
+								opened={ isSearching || isPanelOpen( category.slug ) }
```

Comment:
I guess we could want this to be considered as part of the implementation of `isPanelOpen`. Small thing.

Reply:
Since `isPanelOpen` is used elsewhere, it seemed a bit cleaner to keep it separate.